test/v2/models/responses/hateoas/HateoasDepartureMessageIdsResponseSpec.scala

```diff
@@ -69,12 +84,20 @@ class HateoasDepartureMessageIdsResponseSpec extends AnyFreeSpec with Matchers w
   }
 
   private def selfUrl(departureId: DepartureId, dateTime: Option[OffsetDateTime]): JsObject = dateTime match {
-    case Some(x) =>
-      val time = DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(x)
+    case Some(odt) =>
+      val time = DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(odt)
       Json.obj(
         "href" -> s"/customs/transits/movements/departures/${departureId.value}/messages?receivedSince=$time"
       )
     case None => Json.obj("href" -> s"/customs/transits/movements/departures/${departureId.value}/messages")
   }
 
+  private def generateResponse(messageId: MessageId) =
+    MessageResponseWithBody(
+      messageId,
+      arbitrary[OffsetDateTime].sample.value,
+      DepartureDeclaration,
+      Some("<CC015C><test>testxml</test></CC015C>")
+    )
```

Comment:
DRY principle, see earlier comment - either make this into a trait-method or ideally use that Arbitrary suggestion from before